docs/modules/deployment/pages/system-scalability/time-series-storage/newts.adoc

```diff
@@ -55,7 +55,7 @@ org.opennms.newts.config.ttl=31540000<6>
 org.opennms.newts.config.resource_shard=604800<7>
 ----
 
-<1> Associate time series data by the foreign ID instead of the database generated Node-ID
+<1> Associate time series data by the foreign ID instead of the database generated Node-ID. 
```

Comment:
Why add a period to this list item and not the rest of them?

Reply:
Oversight. Technically, 1 and 2 should have periods, as they are full sentences with verbs, and then the others should not. However, I think that always looks messy, even though it's grammatically correct. Will have to check new style guide to see what NH style guide says. May revise in future. For now I'll add periods to all of them.